nidx/src/api/shards.rs

```diff
@@ -39,8 +39,8 @@ pub async fn create_shard(
     let mut tx = meta.transaction().await?;
     let shard = Shard::create(&mut *tx, kbid).await?;
     // TODO: support other indexes
-    Index::create(&mut *tx, shard.id, "fulltext", IndexConfig::new_fulltext()).await?;
-    // Index::create(&mut *tx, shard.id, "keyword", IndexConfig::new_keyword()).await?;
+    Index::create(&mut *tx, shard.id, "text", IndexConfig::nex_text()).await?;
+    Index::create(&mut *tx, shard.id, "paragraph", IndexConfig::new_paragraph()).await?;
```

Comment:
Yeah, went through the same reasoning. I changed it to be unified with the database at this point, because it was mixed.

I think it would make sense to change the names to match the API maybe. In any case, I hate `text` as an index name, so I'm all for changing it. But since it means touching the DB, let's do it in another PR. I'll leave a TODO here.